packages/world/src/Constants.sol

```diff
@@ -22,3 +22,6 @@ int32 constant MAX_PLAYER_INFLUENCE_HALF_WIDTH = 10;
 int32 constant MAX_PLAYER_RESPAWN_HALF_WIDTH = 10;
 
 address constant ORACLE_ADDRESS = 0xe67AcfFCe0B542540F1520a9eaD7Aa86ff31196E;
+
+int32 constant CHUNK_SIZE = 16;
+int32 constant AREA_SIZE = 512;
```

Comment:
Not used yet but the idea is an area is 512x512 (max height) and consists of many chunks. We commit to an area via a merkle root, and also already set the initial energy. Then as chunks are explored, we verify them via merkle proof to the area's merkle root.